src/Analyzers/MSTest.SourceGeneration/MSTest.SourceGeneration.csproj

```diff
@@ -46,6 +48,7 @@ This package provides the C# source generators for MSTest test framework.]]>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" PrivateAssets="all" />
     <!-- Analyzers to help write analyzers/generators -->
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" PrivateAssets="all" />
+    <PackageReference Include="Polyfill" PrivateAssets="all" />
```

Comment:
This one is likely to cause us issues, if not now, later. This project targets netstandard2.0 and has IVT exposed to test project (not netstandard2.0). So likely going to cause us issues, unless either IVT is removed and make the needed members public, or EmbeddedAttribute is used.